server/src/main/java/org/apache/uniffle/server/ShuffleServerGrpcService.java

```diff
@@ -635,6 +645,15 @@ public void finishShuffle(
       String appId = req.getAppId();
       int shuffleId = req.getShuffleId();
       auditContext.withAppId(appId).withShuffleId(shuffleId);
+      org.apache.uniffle.common.StorageType storageType =
+          shuffleServer.getShuffleServerConf().get(ShuffleServerConf.RSS_STORAGE_TYPE);
+      boolean storageTypeWithMemory =
+          StorageType.withMemory(StorageType.valueOf(storageType.name()));
+      if (storageTypeWithMemory) {
+        LOG.warn(
```

Comment:
We should throw exception directly.

Reply:
@jerqi If we throw exception, it will never support for test mode and `LOCALFILE` client to access the server with `LOCALFILE_MEMORY` storageType?

Reply:
OK for me.

Reply:
I means that it's ok for me to throw exception here.

Reply:
@jerqi Sorry for the misunderstand, done.